cne/cne-converter/src/main/java/com/powsybl/cne/converter/ExportOptions.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.cne.converter;
+
+import java.util.*;
+
+/**
+ * @author Thomas Adam <tadam at silicom.fr>
+ */
+public class ExportOptions {
```

Comment:
This should be renamed more specifically, there is already a `ExportOptions` class in powsybl library, it might be confusing. It can be renamed CneExportOptions.

Reply:
Done